src/client/app/offers/edit_offer.html

```diff
@@ -0,0 +1,160 @@
+          <h2 class="text-left">Нова пропозиція</h2>
+      </div>
+      <form class="form-horizontal">
+          <div class="form-group row">
+              <label class="control-label col-sm-2" for="title" >Назва</label>
+              <div class="col-sm-6">
+                  <!-- here should be pre-filled data from previous page -->
+                  <input type="text" class="form-control" name="heading" id="title" placeholder="Светр для дитячого будинку" ng-model="vm.editableOfferTittle" ng-show="vm.editorEnabled">
+                  {{vm.offerTittle}}
+              </div>
+          </div>
+          <div class="form-group row">
+              <label class="control-label col-sm-2">Категорія</label>
+              <div class="col-sm-6">
+                  <!-- here should be pre-filled data from previous page -->
+                  <ol class="breadcrumb">
+                      <li>
+                          <a href="#">Одяг</a>
+                      </li>
+                      <li>
+                          <a href="#">Дитячий одяг</a>
+                      </li>
+                      <li>Светри</li>
+                  </ol>
+              </div>
+          </div>
+          <div class="row">
+              <div class="col-sm-8">
+                  <hr>
+              </div>
+          </div>
+          <div class="form-group row">
+              <label class="control-label col-sm-2">Опис пропозиції</label>
+              <div class="row">
+              <div class="col-sm-6">
+                  <div ng-show="vm.editorEnabled">
+                      <textarea class="form-control" rows="5" ng-model="vm.editableOffer" ng-show="vm.editorEnabled"></textarea>
+                    </div>
+                     {{vm.offerText}}
+              </div>
+          </div>
+          </div>
+          <div class="form-group row">
+              <label class="control-label col-sm-2">Фото:</label>
+              <div class="col-sm-6">
+                  <input type="file" id="exampleInputFile">
+              </div>
+          </div>
+          <div class="form-group row">
+              <label class="control-label col-sm-2">Розмір:</label>
+              <div class="col-sm-2">
+                  <select class="form-control" ng-model="vm.editableOfferSize" ng-show="vm.editorEnabled">
+                      <option>S</option>
+                      <option>M</option>
+                      <option>L</option>
+                      <option>XL</option>
+                  </select>
+              </div>
+              {{vm.offerSize}}
+          </div>
+          <div class="row">
+              <div class="col-sm-8">
+                  <hr>
+              </div>
+          </div>
+          <div class="form-group row" >
+              <label class="control-label col-sm-2">Регіон:</label>
+              <div class="col-sm-3">
+                  <select class="form-control" ng-model="vm.editableRegion" ng-show="vm.editorEnabled">
```

Comment:
same, values should be extracted to controller and ng-options should be used to display them.